src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -418,6 +424,15 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
     if (opportunityType === AsymSide.Asset) return walletSupportsAsset
   }, [opportunityType, walletSupportsAsset, walletSupportsRune])
 
+  const isVotingPowerReady = useMemo(() => {
+    // No ETH accountIds in the store - we can't get voting power
+    // Meaning this is either a Ledger without EVM accounts connected, or a wallet that explicitly doesn't support ETH (e.g Keplr)
+    if (!walletSupportsEth) return true
+    if (votingPower !== undefined) return true
+
+    return false
+  }, [votingPower, walletSupportsEth])
+
```

Comment:
Thinking more holistically here, there is a bunch of bugs that could be caused by this scenario, where we expect voting power to be ready, may make sense to add the `ethAccountIds.length` check in `selectVotingPower` instead and simply return a vp of 0 there if no Ethereum AccountIds are present?

Reply:
I think we should definitely do this at a selector level so we don't need to rely on consumers to implement this logic, but this PR is fine to fix the obvious and immediate issue.